modules/tinymce/src/themes/silver/main/ts/ui/header/InlineHeader.ts

```diff
@@ -196,20 +197,25 @@ export const InlineHeader = (
 
   const show = () => {
     visible.set(true);
-    Css.set(outerContainer.element, 'display', 'flex');
+    Css.set(uiRefs.mainUi.outerContainer.element, 'display', 'flex');
     DOM.addClass(editor.getBody(), 'mce-edit-focus');
-    Css.remove(uiMothership.element, 'display');
+    Arr.each(uiRefs.uiMotherships, (m) => {
+      Css.remove(m.element, 'display');
+    });
     updateMode(false);
     updateChromeUi();
   };
 
   const hide = () => {
     visible.set(false);
-    if (uiComponents.outerContainer) {
-      Css.set(outerContainer.element, 'display', 'none');
+    // In what situation would `outerContainer` not be set?
```

Comment:
This looks like something that was left in unintentionally? Either that or the check needs to be removed with this comment or a TODO and Jira to investigate. I suspect this was probably left over from refactoring where at some point this might have been nullable.

Reply:
Yeah, I'm still not 100% sure what I should do when I see code that doesn't make sense to me. I guess my instinct is to tag it with a comment, but that doesn't really help anyone else.

I think in this case, the way the code has been structured now, this will never not be set, so I'll remove the check.

- [ ] remove the check.

Reply:
Removed.